tests/document/indexes.py

```diff
@@ -539,6 +539,36 @@ class BlogPost(Document):
         with self.assertRaises(ValueError):
             BlogPost.objects.hint(("tags", 1)).count()
 
+    def test_collation(self):
+        base = {"locale": "en", "strength": 2}
+
+        class BlogPost(Document):
+            name = StringField()
+            meta = {
+                "indexes": [
+                    {"fields": ["name"], "name": "name_index", "collation": base}
+                ]
+            }
+
+        BlogPost.drop_collection()
+
+        names = tuple("%sag %i" % ("t" if n % 2 == 0 else "T", n) for n in range(10))
```

Comment:
its minor but I find this a bit hard to read. I'd suggest to have an explicit tuple with 4 entries and avoid the inner loop
```
tuple("tag 1", "Tag 2", "tag 3", "Tag 4")
```